tests/unit/session_test.py

```diff
@@ -19,3 +21,11 @@ def test_session_connection_refused(free_port_num):
     ssh_session = Session()
     with pytest.raises(LibsshSessionException, match=error_msg):
         ssh_session.connect(host='127.0.0.1', port=free_port_num)
+
+
+def test_session_log_level():
+    """Test setting the log level."""
+    ssh_session = Session()
+    assert ssh_session
```

Comment:
This doesn't seem necessary. I'd instead expect some log-related checks. Perhaps, using the `caplog` fixture that pytest provides..